css/main.css

```diff
@@ -0,0 +1,631 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&family=Inter:wght@400;700&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  scroll-behavior: smooth;
+}
+
+.hidden {
+  display: none !important;
+}
+
+/* .overflow-hidden {
+  overflow: hidden;
+} */
```

Comment:
- [OPTIONAL] Please consider removing any commented code from your codebase except for comments ⛔. This will help you keep your code clean, readable, and following the D.R.Y principle.